### DISCUSSION TITLE
Why @storybook/*, @babel/*, @types/* present in dependencies and not in devDependencies?

### DISCUSSION THREAD

**Discussion by HarshadAgnihotri** [storybookjs/storybook#18528] [Help] [answered]
In storybook/app/react, the components @storybook/*, @babel/*, @types/* are present in dependencies which ideally should be in devDependencies. Can anyone help to answer why these components exists in dependencies?

**Comment by shilman** [comment]
In theory, they are regular dependencies when the types are publicly visible to consumers of the package, and devDependencies when they are only used internally to the package. @ndelangen @tmeasday @yannbf  is there any way to test this?

**Comment by ndelangen** [comment, marked as answer]
If the dependencies can be used at runtime by the user, they should be regular dependencies.
This (potentially) is the case with all package you listed.
